phoenix-core/src/main/java/org/apache/phoenix/schema/tool/SchemaExtractionProcessor.java

```diff
@@ -396,6 +400,35 @@ private TableDescriptor getTableDescriptor(ConnectionQueryServices cqsi, PTable
         }
     }
 
+    private String convertColumnQualifiersToString(PTable table) {
+        StringBuilder cqBuilder = new StringBuilder();
+        if (shouldGenerateWithDefaults)
```

Comment:
I won't add more comments, but make sure you don't use ifs without braces anywhere.